Description/en-US.md

```diff
@@ -1,75 +1,65 @@
+<!-- Use your language name, in your language as the title. Append the country in parentheses, if your language has regional variants. -->
 # English (US)
+<!-- Since EN-US is supposed to be the "default" description, omit this header when pasting to distribution platforms -->
 
-A Fabric-based modpack for Minecraft that focuses on performance and graphics optimizations. 
+A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+<a href="https://www.youtube.com/watch?v=bb8G9X5Q_4I"><img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - A YouTube showcase by Rexus</a>
```

Comment:
```suggestion
<a href="https://www.youtube.com/watch?v=bb8G9X5Q_4I"><img src="https://img.youtube.com/vi/bb8G9X5Q_4I/mqdefault.jpg" alt="Current review" height="140"/> - A YouTube showcase by Rexus</a>
```
A smaller image is possible to get from YouTube directly, without having to downscale the big one. See also: https://tubeskills.com/youtube-thumbnail-grabber/